ddtrace/bootstrap/sitecustomize.py

```diff
@@ -102,22 +101,37 @@ def update_patched_modules():
 
     # Ensure sitecustomize.py is properly called if available in application directories:
     # * exclude `bootstrap_dir` from the search
-    # * find a user `sitecustomize.py` module
-    # * import that module via `imp`
+    # * find and import user `sitecustomize.py` module
     bootstrap_dir = os.path.dirname(__file__)
-    path = list(sys.path)
 
-    if bootstrap_dir in path:
-        path.remove(bootstrap_dir)
+    if sys.version_info < (3, 3, 0):
```

Comment:
Why using two different paths? I'd imagine the second version would work for Python 2 too?
(if not maybe write a comment as why)

Reply:
Yeah it doesn't work in Python 2 for some reason, have to look into why 🤔

Reply:
So I did some digging. It doesn't work in Python 2 is because deleting `sys.modules[<current module>]` makes all references `None` (looks like it gets gc'd immediately). It appears that this can be prevented by keeping a reference to the module before it's removed from `sys.modules`